dist/assests/css/header-banner-nav.css

```diff
@@ -0,0 +1,161 @@
+@import url('header-banner-nav.css');
+/* Main Section, Header, Nav */
+.main-header {
+  background-image: url('../img/banner.jpg');
+  height: 100vh;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  width: 100%;
+}
+
+.main-header .main-header__nav {
+  /* background-color: blue; */
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  width: 100%;
+  margin-right: 1rem;
+  font-size: 11px;
+  font-weight: 600;
+  letter-spacing: 0.02em;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 40;
+}
+.main-header__nav .left {
+  margin-left: 2rem;
+}
+.main-header__nav .right {
+  margin-right: 1rem;
+}
+.main-header__nav ul,
+.main-header__nav ul li {
+  display: inline-block;
+  padding: 0.8rem 1rem;
+}
+.main-header__nav ul,
+.main-header__nav ul li a {
+  text-decoration: none;
+  color: white;
+}
+.logo a {
```

Comment:
in the original mockups (the following pic) 
Original Mockups
![image](https://user-images.githubusercontent.com/1198975/79081108-9f181a00-7d1a-11ea-8555-641081f379cd.png)

This PR implementation
![image](https://user-images.githubusercontent.com/1198975/79081103-9889a280-7d1a-11ea-85f9-8fbb1e5ac8ba.png)


the logos is vertically centered with the left navigation, it would be nice if you can fix the alignement

Reply:
okk working on it

Reply:
Done